src/test/java/edu/unc/lib/boxc/migration/cdm/services/ArchivalDestinationsServiceTest.java

```diff
@@ -138,4 +139,129 @@ public void collectionNumberToPidTest() throws Exception {
         assertEquals("resourceType:Collection",
                 Arrays.stream(solrValues.get(0).getFilterQueries()).findFirst().get());
     }
+
+    @Test
+    public void newDestinationsFileWithPid() throws Exception {
+        solrResponseWithPid();
+        Path destinationMappingsPath = project.getDestinationMappingsPath();
+
+        var options = new DestinationMappingOptions();
+        options.setFieldName("dmrecord");
+
+        service.addArchivalCollectionMappings(options);
+        assertTrue(Files.exists(destinationMappingsPath));
+
+        try (
+                Reader reader = Files.newBufferedReader(destinationMappingsPath);
+                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT
+                        .withFirstRecordAsHeader()
+                        .withHeader(FieldUrlAssessmentService.URL_CSV_HEADERS)
+                        .withTrim());
+        ) {
+            List<CSVRecord> rows = csvParser.getRecords();
+            assertEquals("", rows.get(0).get(1));
```

Comment:
I think the outcome would be a lot clearer if we asserted all the values per row (since there's only 3). We could do something like:
`assertIterableEquals(Arrays.asList("dmrecord:604", "", "604"), rows.get(0));`
i didn't initially understand that the next assertion was expecting a collection id value in the pid column. It'd be good to update the other tests to work this way too